src/app/collective-rewards/rewards/backers/BackerRewardsTable.tsx

```diff
@@ -40,14 +41,18 @@ const tableHeaders: TableHeader[] = [
     tooltip: {
       text: (
         <>
-          An estimate of this Cycle’s rewards from each Builder that will become fully claimable by the end of
-          the current Cycle. These rewards gradually become claimable and are added to your ‘Claimable
-          Rewards’ as the cycle progresses. To check the cycle completion, go to Collective Rewards → Current
-          Cycle.
-          <br />
-          <br />
-          The displayed information is dynamic and may vary based on total rewards and user activity. This
-          data is for informational purposes only.
+          <Typography>
```

Comment:
Since you are using Typography, Im getting the following error:

```
hook.js:608 In HTML, <p> cannot be a descendant of <p>.
This will cause a hydration error.
```

The error is longer but you can check it in the console.

Reply:
aaaah this.... bloody annoying that is
thanks

Reply:
done